### PR TITLE
🇿 Tickets.. improvements and fixes

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: services/tickets/zendesk/service.go [hunk 1]
```diff
@@ -64,21 +67,14 @@ func (s *service) Open(session flows.Session, subject, body string, logHTTP flow
...
 			ExternalID: string(session.Contact().UUID()),
 			Name:       contactDisplay,
 		},
-		DisplayInfo: []DisplayInfo{
-			{
-				// need to also include our ticket UUID in here so the Zendesk app can access it
-				Type: "temba",
-				Data: map[string]string{"uuid": string(ticketUUID)},
-			},
-		},
```
Comment (rowanseymour) [inline services/tickets/zendesk/service.go:81]: was using this to get ticket UUID in app but now we're calling zendesk api to set ticket UUID as external ID

File: web/testing.go [hunk 2]
```diff
@@ -46,6 +46,7 @@ func RunWebTests(t *testing.T, truthFile string) {
 		HTTPMocks    *httpx.MockRequestor `json:"http_mocks,omitempty"`
 		Method       string               `json:"method"`
 		Path         string               `json:"path"`
+		Headers      map[string]string    `json:"headers,omitempty"`
```
Comment (rowanseymour) [inline web/testing.go:49]: need this so some tests can pass Basic Auth headers

File: models/assets.go [hunk 3]
```diff
@@ -283,6 +283,7 @@ func NewOrgAssets(ctx context.Context, db *sqlx.DB, orgID OrgID, prev *OrgAssets
 		}
 	} else {
 		o.ticketers = prev.ticketers
+		o.ticketersByID = prev.ticketersByID
```
Comment (rowanseymour) [inline models/assets.go:286]: oops.. just a bug

File: hooks/ticket_opened.go [hunk 4]
```diff
@@ -61,7 +62,7 @@ func handleTicketOpened(ctx context.Context, tx *sqlx.Tx, rp *redis.Pool, org *m
 		event.Ticket.Body,
 		map[string]interface{}{
 			"contact-uuid":    scene.Contact().UUID(),
-			"contact-display": scene.Contact().Format(org.Env()),
+			"contact-display": tickets.GetContactDisplay(org.Env(), scene.Contact()),
```
Comment (rowanseymour) [inline hooks/ticket_opened.go:65]: need a consistent non-empty way to refer to the contact that respects things like URN redaction

**Review by nicpottier — APPROVED** [review]

**Pull request merged**